docs/content/docs/custom-resource/reference.md

```diff
@@ -251,6 +261,7 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 | startTime | java.lang.String | Start time of the job. |
 | updateTime | java.lang.String | Update time of the job. |
 | savepointInfo | org.apache.flink.kubernetes.operator.crd.status.SavepointInfo | Information about pending and last savepoint for the job. |
+| jobHealthInfo | java.lang.String | Information about job health. |
```

Comment:
I have been thinking a bit about this, we currently have a `clusterInfo` field in the `FlinkDeploymentStatus` we could simply use that map to store this information. 

I think originally it was designed with some flexibility in mind so we can store different metadata. cc @morhidi

Reply:
That's a key-value store which is designed to store config info:
```
    /** Config information from running clusters. */
    private Map<String, String> clusterInfo = new HashMap<>();
```
For example one key is `DashboardConfiguration.FIELD_NAME_FLINK_VERSION`.
I can put it there but at the first glance it's not intended to store structures like this.
Are we sure that we want to do that?

Reply:
OK, moved the health info to the `clusterInfo` map.